user_visit/models.py

```diff
@@ -82,6 +85,21 @@ class UserVisit(models.Model):
         help_text=_lazy("Client User-Agent HTTP header"),
         blank=True,
     )
+    browser = models.CharField(
+        max_length=200,
+        blank=True,
+        null=True,
```

Comment:
An empty string is preferred to `null` so we only have to deal with a single falsey value.
```suggestion
        default="",
```